packages/frontend/src/lib/sets.ts

```diff
@@ -148,6 +152,12 @@ export const SET_METADATA: { [key: string]: SetMetadata } = {
     description: 'Was one of the first 500 beacon chain genesis stakers',
     displayName: 'First 500 Beacon Chain Genesis Staker',
   },
+  'protocol-guild': {
+    count: 152,
+    duneURL: 'https://app.splits.org/accounts/0x84af3D5824F0390b9510440B6ABB5CC02BB68ea1/',
+    description: 'Protocol Guild',
+    displayName: 'Protocol Guild',
```

Comment:
can you change displayName to include snapshot block for now? 

we can make it a mutating merkle tree later

Reply:
added!